lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
Thanks Richard. Could you please make those changes in the developer2 branch rather than on master, so that we don't have to do the work twice.

Is ess-rgui.el still relevant at all for using ESS on Windows? If it's possible to interact with the R process the normal way, I think we should drop support for RGUI interaction, especially considering the limitations you just exposed.